Added documentation for reordering personal tabs (#2633)
MicrosoftDocs/msteams-docs: msteams-platform/tabs/how-to/create-tab-pages/content-page.md

Ensure that the all URL domains used in your tabs are included in the `validDomains` array in your [manifest](~/concepts/build-and-test/apps-package.md). For more information, see [validDomains](~/resources/schema/manifest-schema.md#validdomains) in the manifest schema reference. However, be mindful that the core functionality of your tab exists within Teams and not outside of Teams.

## Reorder static personal tabs

> Starting with manifest version 1.7, developers can rearrange all tabs in their personal app. In particular, a developer can move the “bot chat” tab (which has always defaulted to the first position) anywhere in the personal app tab header. We’ve declared two reserved tab entityId keywords: “conversations” and “about”.

Reviewer: ```suggestion
Starting with manifest version 1.7, developers can rearrange all tabs in their personal app. In particular, a developer can move the *bot chat* tab, which always defaults to the first position, anywhere in the personal app tab header. We’ve declared two reserved tab entityId keywords, *conversations* and *about*.
```

> If you create a bot with a “personal” scope, it will show up in the first tab position in a personal app by default. If you wish to move it to another position, you need to add a static tab object to your manifest with the reserved keyword “conversations”. Wherever you add the “conversations” tab in the “staticTabs” array, that’s where the conversation tab will appear on web and desktop. 

Reviewer: ```suggestion
If you create a bot with a *personal* scope, it will show up in the first tab position in a personal app by default. If you wish to move it to another position, you must add a static tab object to your manifest with the reserved keyword, *conversations*. The *conversation* tab appears on web or desktop based on where you add the *conversation* tab in the `staticTabs` array. 
```

Reviewer: The link is seen as text in the **Integrate your code with Teams** section. Please refer the following image. Request you to resolve the same.
![image](https://user-images.githubusercontent.com/70375155/106588184-7f373880-6570-11eb-811b-173d947464ca.png)
